### PR TITLE
Add election timetable to Election pages

### PR DESCRIPTION
**Pull request by symroe**
A quick implementation of the new `.timetable` method of `uk-election-timetables`.

Screenshot from 2021-02-11 11-11-07

### CODE REVIEW

File: every_election/apps/elections/models.py [hunk 1]
```diff
@@ -280,6 +282,25 @@ def get_example_postcode(self):
...
 
+    @property
+    def get_timetable(self):
+        country_map = {
+            "WLS": Country.WALES,
+            "ENG": Country.ENGLAND,
+            "NIR": Country.NORTHERN_IRELAND,
+            "SCT": Country.SCOTLAND,
+        }
+        area = self.division or self.organisation
```
Comment (GeoWill) [inline every_election/apps/elections/models.py:293]: TIL you can assign like this in python. Must've seen it before, but never clicked.

Comment (symroe) [reply every_election/apps/elections/models.py:293]: Actually I've never used this much, but I was inspired by @michaeljcollinsuk's PR here: https://github.com/DemocracyClub/WhoCanIVoteFor/commit/42968771c1b0c2b3a379287ccb3fcdae43ef40d1#diff-8e7e5cffbf2702ce7fa6dbe7ccaf8383a8ee8094d04bbf5879327c13f2fa4720R95

File: every_election/apps/elections/models.py [hunk 2]
```diff
@@ -280,6 +282,25 @@ def get_example_postcode(self):
...
+
+        if area:
+            territory_code = area.territory_code or self.organisation.territory_code
+            if territory_code:
+                timetable = from_election_id(
+                    self.election_id, country=country_map[territory_code]
+                ).timetable
+                return sorted(timetable, key=lambda r: r["date"])
```
Comment (GeoWill) [inline every_election/apps/elections/models.py:301]: Am I right in thinking https://github.com/DemocracyClub/uk-election-timetables/pull/11 will make the `sorted` unnecessary?

Comment (symroe) [reply every_election/apps/elections/models.py:301]: Yes - unnecessary, but also it won't matter to be sorted twice. Added here and then decided to upstream it

File: every_election/apps/elections/models.py [hunk 3]
```diff
@@ -280,6 +282,25 @@ def get_example_postcode(self):
...
+    def get_timetable(self):
+        country_map = {
+            "WLS": Country.WALES,
+            "ENG": Country.ENGLAND,
+            "NIR": Country.NORTHERN_IRELAND,
+            "SCT": Country.SCOTLAND,
+        }
+        area = self.division or self.organisation
+
+        if area:
```
Comment (michaeljcollinsuk) [inline every_election/apps/elections/models.py:295]: I tend to prefer to use an inverted conditional/return early pattern to help avoid nesting if statements like this

```
if not area:
    return None

territory_code = area.territory_code or self.organisation.territory_code
if not territory_code:
    return None

timetable = from_election_id(
    self.election_id, country=country_map[territory_code]
).timetable
return sorted(timetable, key=lambda r: r["date"])
```

Comment (symroe) [reply every_election/apps/elections/models.py]: Good call, I've made this change

**Comment by symroe** [comment]
Updated to `2.1.1` and removed the sorting - 2.1.1 adds sorting by default

**Review by michaeljcollinsuk — APPROVED** [review]

**Pull request merged**